src/flow/common.js

```diff
@@ -1,18 +1,26 @@
-import { createElement } from 'react'
-import { styled, setup } from 'goober'
+import React from 'react'
+import PropTypes from 'prop-types'
 
+import { makeStyles } from '@material-ui/core/styles'
 
-setup(createElement)
 
-export const Dot = styled('span')`
-  height: 0.5rem;
-  width: 0.5rem;
-  background-color: ${({ state }) => ({
-    failed: '#ea0000;',
-    success: '#00d308;',
-    running: '#009aff;',
-    skipped: '#eeeeee;',
-  }[state] || '#eeeeee')};
-  border-radius: 50%;
-  display: inline-block;
-`
+const useStyles = makeStyles((theme) => ({
+  dot: ({ state }) => ({
+    height: '0.5rem',
```

Comment:
I'd use fixed value (`4px` or `theme.spacing(0.5)`) as a base MUI font size in Snoke is different than the default one. Otherwise, it'll probably distort the proportion.

Reply:
`spacing(1)` is used as the dot is intended to be approximately the same size as the base font-size (or else it'd be too smal)